pdsfile/pds4file/rules/uranus_occs_earthbased.py

```diff
@@ -265,7 +265,7 @@
          r'bundles/\1/uranus_occ_support/document/user_guide/*-user-guide.pdf',
          r'bundles/\1/uranus_occ_support/document/user_guide/*-user-guide.xml',
          r'bundles/\1/uranus_occ_support/document/user_guide/*.pro',
-         r'bundles/\1/uranus_occ_support/document/user_guide/*.py',
+         r'bundles/\1/uranus_occ_support/document/user_guide/*.txt',
```

Comment:
There are no `*.txt` files there. `*.py` was correct. The Python programs are part of the documentation.